shell/platform/darwin/common/framework/Headers/FlutterCodecs.h

```diff
@@ -230,7 +230,7 @@ FLUTTER_DARWIN_EXPORT
 /**
  * The method name.
  */
-@property(readonly, nonatomic) NSString* method;
+@property(readonly, nonatomic, copy) NSString* method;
```

Comment:
Technically it is a breaking change.

Though I don't think making it a "copy" it is going to change the way the public API is actually used unless it was abused where someone intentionally make it mutable? 

Thoughts?

Reply:
Since it's readonly, the setter is not synthesized, so it's not a breaking change here.

Reply:
The breaking part I'm talking about is when someone assigns it to a MutableString, then changes the mutableString and this property will be changed. 

So it was originally a bug. By "breaking". I meant if someone is abusing this bug to "set" the value of these properties, their code will stop working.

Reply:
After another look, I realized I was mixing stuff in my head. I think @hellohuanlin said was right. And since the setters are not synthesized, copy doesn't even make sense here. I will revert these changes.